client/src/app.tsx

```diff
@@ -44,14 +44,21 @@ function Layout({ pageType, children }) {
   React.useEffect(() => {
     setCategory(docCategory({ pathname }));
   }, [pathname]);
+
   return (
     <>
       <A11yNav />
       {!isServer && hasActiveBanners && <Banner />}
-      <div className={`page-wrapper  ${category || ""} ${pageType}`}>
-        <TopNavigation />
-        {children}
-      </div>
+      {pageType === "document-page" ? (
+        <div className={`page-wrapper  ${category || ""} ${pageType}`}>
+          {children}
+        </div>
+      ) : (
+        <div className={`page-wrapper  ${category || ""} ${pageType}`}>
+          <TopNavigation />
+          {children}
+        </div>
+      )}
```

Comment:
This can be deduplicated:

```suggestion
      {<div className={`page-wrapper  ${category || ""} ${pageType}`}>
          (pageType === "document-page" && <TopNavigation />)
          {children}
        </div>}
```

Reply:
Fixed in 78378b823.